rituals/retros.md

```diff
@@ -0,0 +1,83 @@
+# Project Retrospectives
+Retrospectives are an integral part of the “inspect and adapt” process. One of the principles of the Agile Manifesto is:
+
+> At regular intervals, the team reflects on how to become more effective, then tunes and adjusts its behavior accordingly.
+
+## Purpose
+The goal of a retrospective is to review the team's process and discuss whether it needs to be improved. It's about actions and outcomes, not an airing of grievances.
+
+Although a good team will always be looking for improvement opportunities, the team should set aside a brief, dedicated period at the end of each project (or each sprint in a longer project) to deliberately reflect on how they are doing and to find ways to improve.
+
+During a retro, the team should feel empowered to shape their process and free to speak about where it failed and how to improve.
+
+### Who should attend the retro?
+Everyone who contributed to the work. At a minimum, that includes the **engineers**, the **product owner**, and the **designers**.
+
+Typically, managers are not invited to retros, because they can also have an unintentionally suppressive effect on open and honest dialogue.
```

Comment:
Interesting! I haven't heard this before. Given how much impact a manager has on the functioning of a team, is there any method for them to be clued in to the conclusions of a retro? It seems like you'd want them to know if the team is having problems with work processes.

Reply:
Two responses:

1) That line may not be appropriate for a company as tight-knit as Cloud Four. I copied this doc from a previous company with a much larger engineering team and a bit more politics.

2) If you're at a company where you need this rule, the manager _absolutely_ needs to be clued into the **results** of the retro. But if you don't feel like you can say "I really struggled to complete my deadlines this week because (political reason)" without getting in trouble — or if your manager is the one causing problems — then you can end up not being able to speak honestly about the challenges facing the team.

But yeah, I don't think those kinds of issues are likely to apply here. I'm happy to remove that line.